src/main/features/certificateOfService/index.ts

```diff
@@ -37,7 +37,7 @@ export class Feature {
           return new DraftCertificateOfService().deserialize(value)
         }))
 
-    app.all(/^\/legal\/certificateOfService\/(what-documents|document-upload|file-upload)$/,
+    app.all(/^\/legal\/certificateOfService\/(what-documents|document-upload|file-upload|document-remove)$/,
```

Comment:
this is getting bigger, is there any page you don't want to load draft for? otherwise use *, follow the citizen changes recently done by Tim

Reply:
the rest of the certificate of service journey will not need this draft. it is only used for file upload